samples/BoltOn.Samples.Application/Entities/Student.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using BoltOn.Cqrs;
+using BoltOn.Samples.Application.Handlers;
+
+namespace BoltOn.Samples.Application.Entities
+{
+	public class Student : BaseCqrsEntity
+	{
+		public string FirstName { get; private set; }
+		public string LastName { get; private set; }
+		public int StudentTypeId { get; private set; }
+
+		private Student()
+		{
+		}
+
+		internal Student(CreateStudentRequest request, string studentType)
```

Comment:
@gokulm : does this have to be internal ? what if the consumer of this bolton doesnt have these entities in application layer? then they will just mark it as public then!

Reply:
@skankanvar yes, just because the handlers are also in the application layer, i declared the ctor internal. if we have the domain entities in a separate domain project, this should be public